src/openklant/components/contactgegevens/migrations/0004_alter_organisatie_adres_nummeraanduiding_id_and_more.py

```diff
@@ -0,0 +1,99 @@
+# Generated by Django 4.2.15 on 2025-01-09 09:22
```

Comment:
Could you add a test for this migration?

Something like https://github.com/maykinmedia/open-klant/blob/f778cd5790d4eadcf3fb378fb4bd16524e5a6b6e/src/openklant/components/token/tests/test_migrations.py#L53